src/capture/capture_video.cpp

```diff
@@ -68,13 +68,7 @@ static ZMBV_FORMAT to_zmbv_format(const PixelFormat format)
 	case PixelFormat::Indexed8: return ZMBV_FORMAT::BPP_8;
 	case PixelFormat::RGB555_Packed16: return ZMBV_FORMAT::BPP_15;
 	case PixelFormat::RGB565_Packed16: return ZMBV_FORMAT::BPP_16;
-
-	// ZMBV is "the DOSBox capture format" supported by external tools such
-	// as VLC, MPV, and ffmpeg. Because DOSBox originally didn't have 24-bit
-	// colour, the format itself doesn't support it and treats 4-byte
-	// formats as byte-ordered BGR colour values.
-	//
-	case PixelFormat::BGR24_ByteArray: return ZMBV_FORMAT::BPP_32;
```

Comment:
So this is no longer true then? Maybe an explanatory comment is in order?

Reply:
:+1: ; added a note in the code.